community/blog/2023-05 New Azure Logic Apps and DevOps functionality in milestone v1.0 Arcus Scripting release/README.md

```diff
@@ -0,0 +1,71 @@
+
+```powershell
+PS> Resubmit-FailedAzLogicAppRuns `
+  -ResourceGroupName "rg-common-dev" `
+  -LogicAppName "rcv-shopping-order-sftp" `
+  -StartTime "2023-05-01 00:00:00" `
+  -EndTime "2023-05-01 10:00:00"
+# Successfully resubmitted all failed instances for the Azure Logic App 'rcv-shopping-order-sftp' in resource group 'rg-common-dev' from '2023-05-01 00:00:00' and until '2023-05-01 10:00:00'
+```
+
+🔗 For more information on this particular command and other Logic Apps-related commands in this module, [see our feature documentation](https://scripting.arcus-azure.net/Features/powershell/azure-logic-apps).
+
+### Cancel running instances on an Azure Logic App
+Same as with failed Logic Apps instances, there is no 'batch' functionality to cancel running instances on an Azure Logic App in the Azure Portal. This is particularly a problem when the instances were accidentally triggered with a big load. With a combination of several Azure PowerShell commands, you can mimic this functionality but it is not maintainable or easy to transfer to other projects that require the same or similar functionality. Arcus Scripting v1.0 also includes a 'batch' version of canceling many running Azure Logic Apps instances in one go.
+
+```powershell
+PS> Cancel-AzLogicAppRuns `
+  -ResourceGroupName "rg-common-dev" `
+  -LogicAppName "rcv-shopping-order-sftp"
+# Successfully cancelled all running instances for the Azure Logic App 'rcv-shopping-order-sftp' in resource group 'rg-common-dev'
+```
+
+🚩 It is important to note that this functionality is externally requested by a contributor. It is yet another a great example of how the usability of the Arcus Scripting library. It is fully grown out of the frustration and struggles of interacting with Azure-related resources using PowerShell. An extra layer that makes this interaction more comfortable and logical.
+
+🔗 For more information on this particular command and other Logic Apps-related commands in this module, [see our feature documentation](https://scripting.arcus-azure.net/Features/powershell/azure-logic-apps).
+
+## Azure DevOps
+There is not even an Azure DevOps package that lets you interact with DevOps-related custom functionality. Interacting with DevOps functionality requires constant research for the correct syntax and possibilities. Luckily, with Arcus Scripting we have bundled commonly used functionality in the `Arcus.Scripting.DevOps` module which the v1.0 release enhances even further. 
```

Comment:
```suggestion
Interacting with DevOps functionality requires constant research for the correct syntax and possibilities. Luckily, with Arcus Scripting we have bundled commonly used functionality in the `Arcus.Scripting.DevOps` module which the v1.0 release enhances even further. 
```

Removed the line about there not being a devops package. There is one for az cli, just not powershell 😉

Reply:
Hehe, yes, good catch. 😄